eclipse-collections-code-generator/src/main/resources/api/map/mutablePrimitivePrimitiveMap.stg

```diff
@@ -56,20 +68,88 @@ public interface Mutable<name1><name2>Map extends <name1><name2>Map, Mutable<nam
      */
     void updateValues(<name1><name2>To<name2>Function function);
 
+    /**
+     * Removes the mapping associated with the key, if one exists, from the map.
+     * @param key the key to remove
+     * @see #remove(<type1>)
+     */
     void removeKey(<type1> key);
 
+    /**
+     * Removes the mapping associated with the key, if one exists, from the map.
+     * @param key the key to remove
+     * @see #removeKey(<type1>)
+     */
     void remove(<type1> key);
 
+    /**
+     * Removes the mapping associated with the key, if one exists, from the map,
+     * returning the previously associated value with the key. If no mapping
+     * existed for the key, the specified default value is returned.
+     * @param key the key to remove
+     * @param value the default value to return if no mapping for the key exists
+     * @return the value previously associated with the key, if one existed,
+     * or {@code value} if not
+     */
     <type2> removeKeyIfAbsent(<type1> key, <type2> value);
 
+    /**
+     * Retrieves the value associated with the key if one exists; if it does not,
+     * associates a value with the key.
+     * a new value with they key
+     * @param key the key
+     * @param value the value to associate with {@code key} if no such mapping exists
+     * @return the value associated with key, if one exists, or {@code value} if not
+     */
     <type2> getIfAbsentPut(<type1> key, <type2> value);
 
+    /**
+     * Retrieves the value associated with the key if one exists; if it does not,
+     * associates the result of invoking the value supplier with the key.
+     * @param key the key
```

Comment:
I understand this better after reading your comment from before but could still be confusing.